Writerside/topics/ranks-overview.md

```diff
@@ -0,0 +1,63 @@
+# Rangübersicht
+
+Auf dem Server gibt es verschiedenste Ränge, welche den Spielern zugeordnet sind.
+Was er zu bedeuten haben und welche Aufgabe sie erfüllen erfährst du auf dieser Seite.
+
+<tabs>
+
+<tab title="Spieler Ränge" >
+
+| Rang | Bedeutung & Erklärung |
+|------|------------------------|
+| ![Contributor](contributor.png) | Verteilt an ehemalige Teammitglieder oder Spieler, die uns auf andere Weise sehr geholfen haben. |
+| ![Veteran](veteran.png) | Verteilt an alle Spieler, welche auf dem 1.19 Server mehr als `200` Stunden aktive Spielzeit hatten. |
+| ![Spieler](player.png) | Der normale Rang für jeden neuen Spieler. |
+</tab>
+
+<tab title="Team Ränge" >
+
+| Rang | Bedeutung & Erklärung |
+|------|------------------------|
+| ![Management](management.png) | Verwaltung der Infrastruktur, Entwicklung von Plugins, Teammanagement. |
+| ![Administrator](administrator.png) | Ansprechpartner bei Bugreports / Erstattungen; Entwicklung von Plugins. |
+| ![Entwickler](developer.png) | Unterstützt uns bei der Entwicklung des Servers und / oder Events. |
+| ![Moderator](moderator.png) | Ebenfalls Ansprechpartner bei Fragen, jedoch auch bezüglich der Regeln und dessen Auslegung; übernimmt moderative Aufgaben, wie das Durchsetzen der Regeln. |
+| ![Supporter](supporter.png) | Ansprechpartner für Fragen bezüglich Funktionen/Features des Servers etc. |
+| ![Builder](builder.png) | Unterstützt uns beim Bauen von Maps für z.B. Events oder andere Serverbereiche. |
```

Comment:
Vielleicht kann man auch in die Tabelle eine Liste machen. Es gibt ja zum Beispiel beim Administrator zwei Punkte

Reply:
Du meinst so:

![image](https://github.com/Keviro/community-server-docs/assets/25409956/56cd8d43-ed43-45f6-b45a-9e030c8f1b99)

Das geht allerdings in reinem Markdown nicht, dann muss man doch wieder HTML benutzen

Reply:
Ja und? Also es sieht doch besser aus oder nicht

Reply:
Ja